### ISSUE TITLE
create show notes for mStable podcast

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-planning#532]
We'll need some nice show notes to summarize the mStable podcast a few weeks ago that Griff did 

We would like to borrow some ideas around governance structure for our the GIVpower spec.

Here's the podcast on the Giveth youtube channel - https://www.youtube.com/watch?v=j7HxR8-uHso

Also the current GIVpower (formerly GIVcuration) spec to help with context.
https://docs.google.com/document/d/17mjEcS948OAjScZnXinGquGwdRLpWi-1EjGcTg6MeBI/edit?usp=sharing

**Comment by laurenluz** [comment]
@jahanzaib-jay can you add the doc you're working on here?

Just to clarify, we really need a good summary of the key points of their governance model so there are kind of two parts to this:
- [ ] shownotes to publish as the youtube video description
- [ ] a summary of the key points of their governance model (for internal use)

lmk if you have any questions

**Comment by jahanzaib-jay** [comment]
Show notes:
https://docs.google.com/document/d/1aO2lm27HL6n_M0u0J73vRenKaU3pRSUBZS3vdyewMyk/edit?usp=drivesdk

Working on the technical case study. Will be done in a few hours.

**Comment by laurenluz** [comment]
@danelsuga can you QA this one? It's good content, but needs a few adjustments with the grammar

**Comment by laurenluz** [comment]
@jahanzaib-jay can you please post the link to the technical summary? this was always the most important part.

**Comment by laurenluz** [comment]
@jahanzaib-jay 👆

**Comment by jahanzaib-jay** [comment]
Here is the technical summary of the podcast: 
https://docs.google.com/document/d/1CVtqXcXVk5hM5cEeRKHeGsDAv1qDzgxQiwMbIsb2DSM/edit?usp=sharing

@laurenluz

**Comment by danelsuga** [comment]
I also reviewed the shownotes and left a few commentes, but one question I have is what information should be included or not there.

**Comment by jahanzaib-jay** [comment]
Okay. Let's wait for Lauren to give her feedback.
I simply followed the shownote guidelines that was provided and I also took inspiration from some previous shownotes.

**Comment by laurenluz** [comment]
@jahanzaib-jay I reviewed both docs - nice work! and thanks @danelsuga for your awesome edits!

I think the shownotes are ok as is, it would have been nice to have a little more detail, but it's not so critical. I'll add them to the youtube caption.

As for the technical summary - really helpful! You did a good job of summarizing key concepts efficiently, especially for V1. For the V2 part though, I noticed you left of something that I remember as pretty important wrt the functionality: the users stake & lock tokens and then get multipliers on them. the same multiplier is applied to both rewards earned & voting powering. It's been months now since I heard this podcast though so my memory isn't very fresh on it...

I left some edits, have a look and lmk if you have any questions. once that's done, we can call this finished.
